std/internal/math/biguintcore.d

```diff
@@ -52,6 +52,130 @@ import std.range.primitives;
 import std.traits;
 
 private:
+
+// compile-time & run-time dipatchers to allows BigInt CTFE for 32 bit OS.
+
+pragma(inline, true)
+uint multibyteAddSub(char op)(uint[] dest, const(uint)[] src1, const (uint)[] src2, uint carry) pure
```

Comment:
Why did you specify the attributes manually? It's a template and DMD should infer the attributes automatically.
The best practice is to avoid specifying the attributes manually, because then future improvements will propagate.